zuul.d/jobs.yaml

```diff
@@ -721,3 +721,22 @@
     cleanup-run: playbooks/new-tenant-juju/cleanup.yaml
     secrets:
       - serverstack_admin
+
+- job:
+    name: publish-charm
+    description: Publish a charm to charmhub
+    timeout: 1200
+    parent: tox
+    provides: charm
+    run: playbooks/charm/publish.yaml
+    secrets:
+      - serverstack_cloud
+      - charmhub_token
+    vars:
+      zuul_site_mirror_fqdn: 10.245.167.88
+      docker_insecure_registries:
+        - 10.245.167.88:8082
```

Comment:
Magic numbers?

These should probably be variables rather than hard coded into the job; perhaps add the var to the `role/publish-charm/defaults/main.yaml`?  It would be even better if it was in an ENV variable somewhere, but let's at least start with the var.

Reply:
Site specific variables can be configured here [1]. However the zuul-scheduler charm does not support configuring the same. As a temporary fix, added the varibles to ``role/publish-charm/defaults/main.yaml``

[1] https://zuul-ci.org/docs/zuul/latest/configuration.html#attr-executor.variables

Reply:
Please ignore my previous comment as it is more about site wide variables rather than site specific variables.
Created new file site-vars.yaml and passed it to the playbook.